mne/utils/docs.py

```diff
@@ -3130,9 +3130,9 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     suppression.
     When ``method='iir'``, ``phase='zero'`` (default) or
     ``phase='zero-double'`` constructs and applies IIR filter twice, once
-    forward, and once backward (making it non-causal) using filtfilt.
+    forward, and once backward (making it non-causal) using `~scipy.signal.filtfilt`.
```

Comment:
At some point we discussed going back to being explicit on these like
```suggestion
    forward, and once backward (making it non-causal) using :func:`~scipy.signal.filtfilt`.
```
so we could potentially change our `default_role` at a later date

Reply:
Sure, but just to know, what does it changes? It doesn't fail to resolve, with or without the explicit `:func:`, `:meth:`, `:class:`, ...

Reply:
see my comment.  It doesn't render parentheses without the explicit role

Reply:
OK disregard that comment, I misread 

> so we could potentially change our default_role at a later date

And it makes sense to be working for now with `default_role = "py:obj"`.

Reply:
Thanks! I'm reading too much in diagonals.. :see_no_evil: